### DISCUSSION TITLE
Full Screen support for devices running OxygenOS (OnePlus devices)

### DISCUSSION THREAD

**Discussion by cheeezie** [ppy/osu#19737] [Ideas] [answered]
for now in the latest build, osu! doesn't fully fill the screen but rather show a black bar in the notch space on OnePlus phones (even on AOSP roms as well), it would be a cool option to make the game use the notch space on most android systems.
![Screenshot_2022-08-13-17-57-48-55_25a1a32208bbcdc1d450b7aa854bb161](https://user-images.githubusercontent.com/102518950/184499598-35a2e26a-2185-4d82-aba4-7b98ed5e61be.jpg)

**Comment by integerrr** [comment, marked as answer]
#18395 #17769 i.e. this is intended behaviour until improvements can be made

**Reply by cheeezie** [reply]
True, Android was designed that way but the devs can add it as a feature to the apk to make it work, same as most popular games nowadays.

**Reply by cheeezie** [reply]
plus, the game still doesn't have high refresh rate support on OxygenOS, because that system has its own way to enable high refresh rate on apps. (of course enabling it on Android is a bad idea due to how demanding the game is at the moment and can cause overheating.)

**Reply by peppy** [reply]
Read the linked issues. They explain the current situation and the path forward.
